src/migrations/cluster-store/3.6.0-beta.1.ts

```diff
@@ -1,34 +1,57 @@
 // Move embedded kubeconfig into separate file and add reference to it to cluster settings
+// convert file path cluster icons to their base64 encoded versions
 
 import path from "path"
 import { app, remote } from "electron"
 import { migration } from "../migration-wrapper";
-import { ensureDirSync } from "fs-extra"
+import fse from "fs-extra"
 import { ClusterModel } from "../../common/cluster-store";
 import { loadConfig, saveConfigToAppFiles } from "../../common/kube-helpers";
 
 export default migration({
   version: "3.6.0-beta.1",
   run(store, printLog) {
-    const migratedClusters: ClusterModel[] = []
-    const storedClusters: ClusterModel[] = store.get("clusters");
     const kubeConfigBase = path.join((app || remote.app).getPath("userData"), "kubeconfigs")
+    const storedClusters: ClusterModel[] = store.get("clusters") || [];
 
-    if (!storedClusters) return;
-    ensureDirSync(kubeConfigBase);
+    if (!storedClusters?.length) return;
```

Comment:
```suggestion
    if (!storedClusters.length) return;
```
`storedClusters` always an object